src/drivers/mct/cime_config/config_component_cesm.xml

```diff
@@ -323,7 +320,7 @@
 
   <entry id="ROF_NCPL">
     <type>integer</type>
-    <default_value>8</default_value>
+    <default_value>$ATM_NCPL</default_value>
```

Comment:
This changes the coupling frequency for I compsets, and maybe others, which we don't want to do.